tests/console/force_cron_run.pm

```diff
@@ -30,18 +30,27 @@ sub settle_load {
 
 sub run {
     select_console 'root-console';
-
     # show dmesg output in console during cron run
     assert_script_run "dmesg -n 7";
 
     # Make sure there's no load before we trigger one via cron.
     settle_load;
     my $before = time;
-    assert_script_run "bash -x /usr/lib/cron/run-crons", 1000;
+    # "bash -x /usr/lib/cron/run-crons" doesn't work really on a running system with uptime less than 15 minutes
+    # because no daily jobs got created or can be excecuted.
+    # we stop and mask btrfs systemd timers. So keep bash -x /usr/lib/cron/run-crons for SLES 12
+    if (is_sle('<15') || is_leap('<15')) {
+        assert_script_run("bash -x /usr/lib/cron/run-crons", 1000);
+    }
+    else {
+        assert_script_run('rpm -q btrfsmaintenance && systemctl start btrfs-balance.service');
+    }
     record_soft_failure 'bsc#1063638 - review I/O scheduling parameters of btrfsmaintenance' if (time - $before) > 60 && get_var('SOFTFAIL_BSC1063638');
     sleep 3;    # some head room for the load average to rise
     settle_load;
-
+    if !((is_sle('<15') || is_leap('<15'))) {
+        assert_script_run('rpm -q btrfsmaintenance && systemctl stop btrfs-* && systemctl mask btrfs-*');
```

Comment:
Why should we stop the services? They should not even be running at this time. But what happens if you mask services that are currently running?